aperturedb/sqlc/queries/onion.sql

```diff
@@ -0,0 +1,16 @@
+-- name: UpsertOnion :exec
+INSERT INTO onion (
+    private_key, created_at
+) VALUES (
+    $1, $2
+) ON CONFLICT (
+    private_key
+) DO NOTHING;
```

Comment:
yep, it works. As you say there in no difference because we do a select from the application and if there is already a row we do not allow inserting a new one. We can leave it as it is, I changed the test to reflect that we are able to upsert (when it matches what we already have in the db) without any problem.